### ISSUE TITLE
orange receptacle icon

### ISSUE THREAD

**Issue by aahmedxlsolutions** [mapbox/maki#356]
Hello! Thanks for contributing. 

If you are opening an issue about the Maki editor, style guidelines or another Maki-related component, please provide a detailed description and any supporting imagery. If you are making an icon request, please make sure the following items are in your issue. 

#### Icon Request
- organge receptacle icon
-This is symbol for receptacles used in parts of USA ,as Victor Stanley(https://www.connect.victorstanley.com) is placing the receptacles for trash collection on the streets of Baltimore, Florida, Washington Dc, Boston, pittsberg - etc and are expanding to other regions as well we would like to register an icon of their own.
Which can be used to display as symbols for better development use as well.

![orangereceptacle-11](https://cloud.githubusercontent.com/assets/25078299/21888516/a097e7b6-d8ea-11e6-9961-925e069bd2b9.png)
![orangereceptacle-15](https://cloud.githubusercontent.com/assets/25078299/21888517/a0a7184e-d8ea-11e6-9c74-e7a833dd5133.png)

**Comment by natslaughter** [comment]
@aahmedxlsolutions 
Thank you for your submittal and interest in Maki! Maki already has a [waste-basket icon](https://github.com/mapbox/maki/blob/master/icons/waste-basket-15.svg), so we suggest using this particular icon and changing its color using the [maki editor](https://www.mapbox.com/maki-icons/editor/). Best of Luck!

**Comment by aahmedxlsolutions** [comment]
@natslaughter 
1. I tried to access waste-basket  icon using mapbox gl which does not display me the icon. I have the code for your reference in the link below:
http://www.w3schools.com/code/tryit.asp?filename=FBTWKEN2SS5K

I have change the icon to town-hall and it works as expected:
http://www.w3schools.com/code/tryit.asp?filename=FBP97UTSUIAQ

2. I have edited the waste-basket icon on maki editor, after download what is the next process as it provides me .svg file and .json file. How do i use it in the code provided in the sample above.
[all_maki_icons.zip](https://github.com/mapbox/maki/files/710995/all_maki_icons.zip)
